modeshape-jcr/src/main/java/org/modeshape/jcr/mimetype/tika/TikaMimeTypeDetector.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.modeshape.jcr.mimetype;
+package org.modeshape.jcr.mimetype.tika;
```

Comment:
prior to this change, the Tika mime type detection classes were instantiated explicitly by ModeShape based on the given [repository configuration](https://github.com/ModeShape/modeshape/blob/master/modeshape-jcr/src/main/java/org/modeshape/jcr/RepositoryConfiguration.java#L1243)

In other words, these were not exposed on the outside, essentially making them an internal implementation detail. 

Since the repository configuration has not changed in any way, there should be no backward-compatibility concerns because of the package and name change for these classes.